### DISCUSSION TITLE
[python client0.7.0]CANCELLED: client cancelled

### DISCUSSION THREAD

**Discussion by Cross-zc** [apache/skywalking#8364] [Q&A] [answered]
sw-python: 0.7.0
skywalking: 8.9
es: 7.16

Question:
skywalking log:
org.apache.skywalking.oap.server.receiver.profile.provider.handler.ProfileTaskServiceHandler - 115 [grpcServerPool-1-thread-24] ERROR [] - CANCELLED: client cancelled

sky config:
![image](https://user-images.githubusercontent.com/13882651/147656861-e5f85cd6-41a7-4b53-bd85-e5dbbe9fd64d.png)
![image](https://user-images.githubusercontent.com/13882651/147656943-02eaed72-931b-4261-8321-139969dabfdc.png)

sw-python config:
default env

es config:
cluster.name: es-cluster
 
node.name: node-1
node.master: true
node.data: true
 
path.data: /var/lib/elasticsearch
path.logs: /var/log/elasticsearch
 
bootstrap.memory_lock: false
 
bootstrap.system_call_filter: false
 
network.host: 0.0.0.0
 
http.port: 9200
 
discovery.seed_hosts: ["xxx"]
 
cluster.initial_master_nodes: ["xxx"]
 
http.cors.enabled: true
 
http.cors.allow-origin: "*"
-------------------------------------------------------------------------

thanks

**Comment by wu-sheng** [comment, marked as answer]
What is the question? This is a connection disconnected.

**Reply by Cross-zc** [reply]
Thank you for answering my question。

      I use the test case and the data on the web page is 0，There are only [client cancelled ]errors in the log.


my python client code:

import time

def method1():
    time.sleep(2)
    return '1'

def method2():
    time.sleep(2)
    return method1()

def method3():
    time.sleep(2)
    return method2()

if __name__ == '__main__':
    import socketserver
    from http.server import BaseHTTPRequestHandler

    class SimpleHTTPRequestHandler(BaseHTTPRequestHandler):

        def do_POST(self):
            method3()
            time.sleep(0.5)
            self.send_response(200)
            self.send_header('Content-Type', 'application/json')
            self.end_headers()
            self.wfile.write('{"song": "Despacito", "artist": "Luis Fonsi"}'.encode('ascii'))

    PORT = 19090
    Handler = SimpleHTTPRequestHandler

    with socketserver.TCPServer(("", PORT), Handler) as httpd:
        httpd.serve_forever()`



start: sw-python -d run xxx.py

**Comment by little-hang** [comment]
I got the same problem ，so, how did you solve the problem
